Docs/manual-testing

```diff
@@ -0,0 +1,115 @@
+Firefox for iOS Testing
+
+Softvision Mobile QA - Firefox for iOS testing tasks and process
+
+## Release
+
+* **Frequency**: Each RC
+
+* **Tasks performed by the QA team:**
+
+   * Smoke and sanity testing
+   * Exploratory testing
+   * Localization testing
+   * Bug triage
+* **Specific Release testing tasks:**
+   *  none
+* **Feature coverage**: yes
+* **Bug verification coverage**: uplifts
+
+## Beta 
+
+* **Frequency:** every other Beta depending on the number of changes
+* **Tasks performed by the QA team:**
+   * Smoke and sanity testing
+   * Exploratory testing
+   * Bug verification
+* **Specific Beta testing tasks:**
+   * Full functional & UI testing
+   * Full Search testing
+* **Feature coverage**: yes
+* **Bug verification coverage**: uplifts
+
+## Main
+
+* **Frequency**: daily 
+* **Tasks performed by the QA team**
+   * Smoke and sanity testing
+   * Exploratory testing
+   * Bug triage
+* **Specific Nightly testing tasks**:
+   * Feature testing
+   * Test case creation
+   * Bug verification
+* **Feature coverage**: yes
+* **Bug verification coverage**: yes
+
+## Release
```

Comment:
Not sure about this release title again... should it be Detailed information about the task performed, and then below the Device coverage and the rest, Full-funcional and the other titles?

Reply:
The main idea was to have the "Detailed information about the task performed" as a different section and the other ones as sub sections.